Webpack error with exports->default in package.json

First off, thanks for your work on the project!

Got this error when using webpack 5.91.0 on version 1.3.2 of format-css:
`Module not found: Error: Default condition should be last one`

Version 1.3.1 works when the last exports was default and when I manually update the package.json for format-css so the last exports is `default` then webpack works.
```
  "exports": {
    "require": "./dist/format.cjs",
    "types": "./dist/index.d.ts",
    "default": "./dist/format.modern.js"
  },
```

[Webpack documentation](https://webpack.js.org/guides/package-exports/#:~:text=The%20last%20condition%20in%20the%20object%20might%20be%20the%20special%20%22default%22%20condition%2C%20which%20is%20always%20matched.)

--- a/package.json
+++ b/package.json
@@ -5,9 +5,9 @@
   "type": "module",
   "source": "index.js",
   "exports": {
+    "types": "./dist/index.d.ts",
     "require": "./dist/format.cjs",
-    "default": "./dist/format.modern.js",
-    "types": "./dist/index.d.ts"
+    "default": "./dist/format.modern.js"
   },
   "types": "./dist/index.d.ts",
   "main": "./dist/format.cjs",

